metadata_backend/api/handlers.py

```diff
@@ -461,6 +463,32 @@ async def delete_user(self, req: Request) -> Response:
         LOG.info(f"DELETE user with ID {user} was successful.")
         return web.Response(status=204)
 
+    async def mirror_dataset(self, req: Request) -> Response:
+        """Mirror dataset content queried from EGA into the database.
+
+        :param req: POST request
+        :returns: HTTP No Content response
+        """
+        dataset_id = req.match_info["datasetId"]
+        db_client = req.app["db_client"]
+        await spawn(req, self.data_mirroring(dataset_id, db_client))
```

Comment:
this returns a Job https://aiojobs.readthedocs.io/en/stable/api.html#job which we can watch to see if it has finished or not.

I think it is perfectly fine if we move these it its own handler class/ file as not sure it reuses anything from the base handler class.

There we could implement also an endpoint where we would check the status of that job (active, pending or closed)

Reply:
The endpoint was moved to its own handler class in `handlers.py`, but job status check endpoint remains unimplemented.